### ISSUE TITLE
Consistent Cable Cluster Crash

### ISSUE THREAD

**Issue by Solego** [hilburn/AdvancedSystemsManager#60]
Version 0.1.9

Using a cable cluster with both redstone emitter and transforming cable camouflage.  Set up to emit a .5 second redstone pulse every 10 seconds.  On pulse, game will crash with 'Fatal Error Has Occurred' message.  No crash log is generated, and fml-latest log doesn't give any reason for crash, only the following.

[20/8/2015 21:52:37 PM] [Server thread/INFO]: Solego lost connection: TextComponent{text='Disconnected', siblings=[], style=Style{hasParent=false, color=null, bold=null, italic=null, underlined=null, obfuscated=null, clickEvent=null, hoverEvent=null}}
[20/8/2015 21:52:37 PM] [Server thread/INFO]: Solego left the game
[20/8/2015 21:52:37 PM] [Server thread/INFO]: Stopping singleplayer server as player logged out

This error did not occur with a cable cluster containing only a redstone emitter, or only a transforming cable camouflage.
This did occur with both the regular and advanced cable clusters.
The crash happened only on the redstone pulse.
Minecraft crashed whether or not a camouflage event was called in the system manager program.
Two seperate cable clusters, one containing redstone emitter, and one containing transforming cable camouflage attached at different points in the network were able to do their respective tasks without problem.
The crash only occurs if a redstone pulse is emitted.
The cable cluster will function fine if there is no emit redstone event in the program.

**Comment by Solego** [comment]
This error occurs with a Double Sided cable camouflage as well, but not with the standard cable camouflage.

**Comment by Solego** [comment]
I tried this with a cable cluster combining an emitter and a sign updater, and the two worked well together. But when I added the cable camouflage to it, it crashed on pulse again.

**Comment by hilburn** [comment]
Do you have a crash log? failing that a console log might help

**Comment by Solego** [comment]
The only console output I got from the AT Launcher console was what I'd pasted.  Before those entries were just the posts about my giving myself items through NEI during my testing.  No crash logs at all were generated, and none of the log files I do have offer any sort of indication as to WHY the error occurred.  Just that suddenly I was disconnected.  I have never tried running Minecraft with a java console window open, but I'll try to get that working tonight and start crashing again to see what I can get out of that.  Barring that, I did notice that whenever I select the cable cluster in the Emitter window, it looked like a redstone emitter, not a cable cluster.  Whenever I selected it in any other screen, the camouflage update event, the sign updater event, etc, it always looked like the cable cluster.  Perhaps that has something to do with it?  I know the emitter itself changes texture to indicate when it is emitting, and the camouflage cable also changes block textures.  The emitter doesn't change the texture on a cable cluster when it emits, but when adding camouflage blocks, is the block getting texture update code that the emitter is trying to take advantage of?  That's the only cause I can brain up, having no idea how the thing is coded.  Anyway, hope this info helps!  I'm at work, so it will be several hours before I get home and can post a console log.
